Sources/Features/Metrics/Model/UniqueEvent.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+final class UniqueEvent: Event {
+
+    // Backtrace API requires unique event name to be a JSON array
+    internal(set) public var name = [String]()
```

Comment:
It seems to be wrong. The class is `internal` so setter and getter are `internal` as well.

Reply:
Will remove, thank you!